models/user.go

```diff
@@ -0,0 +1,51 @@
+package models
+
+import (
+	"github.com/beego/beego/v2/client/orm"
+)
+
+// User : User model
+type User struct {
+	Base
+
+	Email             string `orm:"unique;size(128)"`
+	EncryptedPassword string `orm:"size(128)"`
```

Comment:
We can use `HashedPassword` name for consistency

Reply:
Update https://github.com/Lahphim/go-crawler-challenge/pull/36/commits/6b4e9e74883bfbabaadfa773e7bc65798de3facf